examples/src/bin/hello_rgb.rs

```diff
@@ -29,13 +31,29 @@ fn main() -> ! {
 
     let io = IO::new(peripherals.GPIO, peripherals.IO_MUX);
 
+    // Each devkit uses a unique GPIO for the RGB LED, so in order to support
+    // all chips we must unfortunately use `#[cfg]`s:
+    cfg_if::cfg_if! {
+        if #[cfg(feature = "esp32")] {
+            let led_pin = io.pins.gpio33;
+        } else if #[cfg(feature = "esp32c3")] {
+            let led_pin = io.pins.gpio2;
+        } else if #[cfg(any(feature = "esp32c6", feature = "esp32h2"))] {
+            let led_pin = io.pins.gpio8;
+        } else if #[cfg(feature = "esp32s2")] {
+            let led_pin = io.pins.gpio18;
+        } else if #[cfg(feature = "esp32s3")] {
+            let led_pin = io.pins.gpio48;
+        }
+    }
+
     // Configure RMT peripheral globally
     let rmt = Rmt::new(peripherals.RMT, 80u32.MHz(), &clocks).unwrap();
```

Comment:
For ESP32-H2 using 80 MHz results in a panic:
```
!! A panic occured in 'src/bin/hello_rgb.rs', at line 51, column 63:
called `Result::unwrap()` on an `Err` value: UnreachableTargetFrequency
```
```suggestion
    #[cfg(not(feature = "esp32h2"))]
    let rmt = Rmt::new(peripherals.RMT, 80u32.MHz(), &clocks).unwrap();
    #[cfg(feature = "esp32h2")]
    let rmt = Rmt::new(peripherals.RMT, 32u32.MHz(), &clocks).unwrap();
```